pytest_tank_test/__init__.py

```diff
@@ -33,7 +33,6 @@ def _initialize_logging():
 
     tank.LogManager().initialize_base_file_handler("tk-test")
     tank.LogManager().initialize_custom_handler()
-    tank.LogManager().global_debug = True
```

Comment:
Having debug logging forced everytime you run the tests with `pytest` is so annoying after using this on a daily basis. Let me decide via `TK_DEBUG` if I want to force it.